migrations/20211002095906-withdrawals.ts

```diff
@@ -0,0 +1,11 @@
+import { Db } from 'mongodb';
+
+export = {
+    async up(db: Db) {
+        return db.collection('withdrawals').updateMany({}, { $set: { rewardId: 0 } });
```

Comment:
@JustinaMary Will this set the rewardId property for all withdrawals in the database to 0? 

If so, thats not correct, since we don't know for what reward that withdrawal was registered. 

I think we should release this feature without a migration since the current data should not change and only new withdrawals will need the rewardId. What do you think?

Reply:
Please remove the mgiration